Detections/AzureDevOps/AzDOAdminGroupAdditions.yaml

```diff
@@ -28,41 +28,14 @@ query: |
   'ProjectC'
   ];
   AzureDevOpsAuditing
-  | where OperationName == "Group.UpdateGroupMembership.Add" and
-    TimeGenerated > ago(timeframe) and
-    Area == "Group" and 
-    (
-      Details contains "was added as a member of group" and 
-      (
-        Details endswith '\\Project Administrators' or 
-        Details endswith "\\Project Collection Administrators"
-      )
-    )
+  | where TimeGenerated >= ago(timeframe)
+  | where Area == "Group" and OperationName == "Group.UpdateGroupMembership.Add"
+  | where Details has 'Administrators'
+  | where Details has "was added as a member of group" and (Details endswith '\\Project Administrators' or Details endswith '\\Project Collection Administrators')
   | parse Details with AddedIdentity ' was added as a member of group [' EntityName ']\\' GroupName
-  | extend 
-    Level = iif(GroupName == 'Project Collection Administrators', 'Organization', 'Project'), 
-    AddedIdentityId = Data.MemberId
-  | extend 
-    Severity = iif(Level == 'Organization', 'High', 'Medium'), 
-    AlertDetails = strcat('At ', TimeGenerated, ' UTC ', ActorUPN, '/', ActorDisplayName, ' added ', AddedIdentity, ' to the ', EntityName, ' ', Level)
-  | where
-    (
-      MontiorAllProjects == true or 
-      EntityName in (ProjectsToMonitor)
-    )
-    or Level == 'Organization'
-  | project 
-    timestamp = TimeGenerated, 
-    Severity, 
-    Adder = ActorUPN, 
-    AddedIdentity, 
-    AddedIdentityId, 
-    AlertDetails, 
-    Level, 
-    EntityName, 
-    GroupName, 
-    ActorAuthType = AuthenticationMechanism, 
-    ActorIpAddress = IpAddress, 
-    ActorUserAgent = UserAgent, 
-    RawDetails = Details
-  | extend AccountCustomEntity = Adder, IPCustomEntity = ActorIpAddress
\ No newline at end of file
+  | extend Level = iif(GroupName == 'Project Collection Administrators', 'Organization', 'Project'), AddedIdentityId = Data.MemberId
+  | extend Severity = iif(Level == 'Organization', 'High', 'Medium'), AlertDetails = strcat('At ', TimeGenerated, ' UTC ', ActorUPN, '/', ActorDisplayName, ' added ', AddedIdentity, ' to the ', EntityName, ' ', Level)
+  | where (MontiorAllProjects == true or EntityName in (ProjectsToMonitor)) or Level == 'Organization'
```

Comment:
Typo - similar to above let. MonitorAllProjects
Not sure about the logic. If MonitorAllProjects == true it will still match to the selected projects. 
Couldn`t test this logic on the actual data.

Reply:
fixed